src/clangd-context.ts

```diff
@@ -68,13 +68,23 @@ export class ClangdContext implements vscode.Disposable {
     if (!clangdPath)
       return null;
 
+    const useScriptAsExecutable =
+        await config.get<boolean>('useScriptAsExecutable');
```

Comment:
I simplified this a bit in https://github.com/clangd/vscode-clangd/pull/731 (`config.get` isn't actually async and doesn't need to be awaited), would you mind rebasing on top of that and querying the config value directly in the constructor?